helpers/sentiment_sort.py

```diff
@@ -0,0 +1,10 @@
+def sort(comment_list):
+    for i in range(1, len(comment_list)):
+        key = comment_list[i]
+        prev_index = i-1
+        while prev_index >= 0 and key.sentiment < comment_list[prev_index].sentiment :
```

Comment:
What is this code doing here?

Reply:
It's sorting the sentiment in the list, so that when quartiles are being found, the data will already be sorted.

Reply:
Do you really need a while inside a for-loop ?